### DISCUSSION TITLE
[Feature Request]: iOS app - Timestamps for messages

### DISCUSSION THREAD

**Discussion by lagunacomputer** [meshtastic/firmware#4526] [Ideas]
[Feature Request]: iOS app - Timestamps for messages

It would be great if the iOS app put a timestamp of some sort on received messages.  Either time message was sent, time message was received by the node, or time message received by the app, doesn't matter.   Case scenario is my node stays upstairs (or i leave the house to go to work for the day), and I don't always have a constant BT connection to the app. I often find myself unnecessarily replying to queued old messages from the day before, once my app reconnects to the node after a few hours .  I imagine thus contributing to unnecessary mesh traffic airtime. I would be great to know if a message is old or not. Thank you for your consideration.

**Comment by garthvh** [comment]
You can long press on a message and see all the details

**Reply by lagunacomputer** [reply]
ok thank you that works for me!
